exercises/practice/reverse-string/.approaches/config.json

```
@@ -0,0 +1,36 @@
+{
+  "introduction": {
+    "authors": [
+      "michalporeba"
+    ]
+  },
+  "approaches": [
+    {
+      "uuid": "0ec50f64-a8f7-49b5-b0d0-3977149c23ba",
+      "slug": "string-builder",
+      "title": "StringBuilder",
+      "blurb": "Using the StringBuilder class.",
+      "authors": [
+        "michalporeba"
+      ]
+    },
+    {
+      "uuid": "9cc22152-82c3-49b1-95a7-8ee62cdeac21",
+      "slug": "its-a-sequence",
+      "title": "It's a sequence",
```

I would simplify the name a bit:

```suggestion
      "slug": "sequence",
      "title": "Use the fact that a string is also a sequence",
```